### PR TITLE
XM Cloud Tutorial: Setup - Split tutorial in two

### PR DESCRIPTION
**Pull request by jst-cyr**
## Description / Motivation
The original tutorial covers both the infrastructure setup (Project/Environment) and Site creation (including connecting your app). This new PR splits that tutorial into two pieces.

## How Has This Been Tested?
Locally and on Vercel

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation update (non-breaking change; modified files are limited to the `/data` directory or other markdown files)

## Checklist:
- [X] I have read the Contributing guide.
- [X] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change.
- [X] My change is a documentation change and there are NO other updates required.
- [ ] My change has new or updated images which are stored in the `/public/images` folder that need to be migrated to Sitecore DAM

### CODE REVIEW

File: apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/create-a-site.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+---
+title: 'Tutorial: How to Create a Site and Connect your App to it'
```
Comment (lovesitecore) [inline apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/create-a-site.md:2]: On my screen it puts the "it" into the next row which does not look good. Maybe title can just be: "Tutorial: How to Create a Site and Connect your App"

**Review by lovesitecore — COMMENTED** [review]
Here is a minor one. Other than that the split looks good and makes the content more accessible.

File: apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/create-a-site.md (hunk 1 above)
Comment (jst-cyr) [reply apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/create-a-site.md]: Great call, and it reads better too. I never liked that "to it" part. Thank you! The commit change is done, deploying it now.

**Comment by markvanaalst** [comment]
FYI; I have updated PR #740 with the latest changes from this branch.

Preview here: [https://developer-portal-git-feature-16596b-sitecoretechnicalmarketing.vercel.app/learn/getting-started/xm-cloud/tutorials/create-a-site](https://developer-portal-git-feature-16596b-sitecoretechnicalmarketing.vercel.app/learn/getting-started/xm-cloud/tutorials/create-a-site)

**Comment by jst-cyr** [comment]
Merging this up so we can keep moving forward

**Pull request merged**